build.sh

```diff
@@ -174,6 +177,8 @@ mkdir_clean ${PUTTY_SRC_DIR}
 unzip -q -d ${PUTTY_SRC_DIR} ${SCRATCH_DIR}/putty-src.zip
 cp ${PUTTY_SRC_DIR}/version.h ${PUTTY_SRC_DIR}/licence.h ${PUTTY_SRC_DIR}/windows/
 
+echo "DEBUG: Printing  MSBuild.exe version..."
```

Comment:
I think it would be better to have "MSBuild.exe" in a constant somewhere before, so that could be used here and in the definition of MSBUILD in line 185.